kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,12 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangowhiteNoise):
+
+    def __init__(self, app):
+        DjangoWhiteNoise.__init__(self, app)
+
+    # Correct some MIME types
+    EXTRA_MIMETYPES = (
+            ('application/font-woff', '.woff'),
+            ('application/font-woff2', '.woff2'),)
```

Comment:
But then, maybe a better approach would be to replace the inner tuple relating to `woff2`, so that we're completely overriding `EXTRA_MIMETYPES` given that it might contain some useful defaults later on. I'll give it a go.